utils/line-directive

```diff
@@ -178,7 +179,10 @@ def map_line_from_source_file(source_filename, source_line_num,
 
 def read_response_file(file_path):
     with open(file_path, 'r') as files:
-        return filter(None, files.read().split(';'))
+        # "Make an iterator out of shlex.shlex.get_token, then consume items
+        # until it returns None." (Then eagerly convert the result to a list so
+        # that we can close the file.)
+        return list(iter(shlex.shlex(files, file_path, True).get_token, None))
```

Comment:
I'd prefer for the `True` value we use a named argument, if that's `shell` it makes it much easier to understand what's going on here.

Reply:
I'd love to as well; unfortunately [shlex doesn't document a name](https://docs.python.org/2/library/shlex.html#shlex.shlex).

Reply:
I believe the name is `posix`, the function signature is `shlex.shlex(instream=None, infile=None, posix=False)` [Lib/shlex.py](https://github.com/python/cpython/blob/491740f116755e220135e596ec802ea3a0f65596/Lib/shlex.py#L23)